smarts/sstudio/types.py

```diff
@@ -372,6 +372,7 @@ class TrapEntryTactic(EntryTactic):
 class UTurn:
     trigger_radius: int = 100
     """This task will be triggered if any vehicles within this radius"""
+    initial_speed: float = 10
```

Comment:
Probably not a good idea to give a default value?

* If not given, use the road speed limit instead.
* A comment for `initial_speed` is also needed.